marklogic-data-hub/src/testFixtures/java/com/marklogic/hub/test/AbstractHubTest.java

```diff
@@ -374,6 +371,27 @@ protected void installUserModulesAndArtifacts(HubConfig hubConfig, boolean force
 
     }
 
+    protected void deployEntityIndexes(HubConfig hubConfig) {
+        EntityManager entityManager = new EntityManagerImpl(hubConfig);
+        entityManager.saveDbIndexes();
+        List<Command> commands = new ArrayList<>();
+
+        Path dir = hubConfig.getEntityDatabaseDir();
+        File finalFile = Paths.get(dir.toString(), HubConfig.FINAL_ENTITY_DATABASE_FILE).toFile();
```

Comment:
Let's not put this here yet, as it only deals with the final file but the method name implies that both staging and final would be updated. I'll add a ticket to the 5.4.0 backend tech debt epic to make it easier to save indexes / deploy them in a test, as a quick search showed that we have a number of tests looking to do this. 

One simplification you can do is this:

dbCommand.execute(newCommandContext());

That avoids having to create a list of commands and a SimpleAppDeployer.

Reply:
Updated to use dbCommand.execute(newCommandContext()) and deploy both staging and final indexes